### PR TITLE
[DONE] Foreach() -> Map()

### PR DESCRIPTION
**Pull request by gijs**
@d-weeteling

### CODE REVIEW

File: src/components/views/DetailView.jsx [hunk 1]
```diff
@@ -139,13 +139,12 @@ class DetailViewComponent extends Component {
...
   }
   formatTabularDataEN(parcel) {
-    let enValue,
-      result = [];
-    Object.keys(GEOSERVER_PARCEL_KEYS).forEach(enKey => {
-      enValue = parcel[enKey] || NO_DATA;
-      result.push({ key: enKey, value: enValue });
+    return Object.keys(GEOSERVER_PARCEL_KEYS).map(enKey => {
+      return {
+        key: GEOSERVER_PARCEL_KEYS[enKey],
```
Comment (gijs) [inline src/components/views/DetailView.jsx]: Or should this be `key: enKey,`?

Comment (d-weeteling) [inline src/components/views/DetailView.jsx]: Yep! Anders krijg je de vietnamese loempia te zien

Comment (gijs) [inline src/components/views/DetailView.jsx]: hehehe

**Comment by gijs** [comment]
Done

**Pull request merged**